### ISSUE TITLE
[doc] Supported OpenStack releases not defined

### ISSUE THREAD

**Issue by cityofships** [openstack-exporter/openstack-exporter#192]
It would be beneficial to specify (eg. in README.md) OpenStack releases that are supported by / tested with versions of openstack-exporter.

Related to #191.

**Comment by niedbalski** [comment]
This is correct and it also goes in line with integration testing that we can run across different versions of OpenStack. Manually testing every release per each supported openstack release isn't something we are doing.

Definitively marking this as something that needs to be addressed.

**Comment by niedbalski** [comment]
@mnaser just asking for some of your expertise on CI here, is there any recommended approach we can follow here to run integration tests across different openstack releases to make sure we're not breaking due to API(s) changing, etc?

**Comment by Sharpz7** [comment]
Closing in favor of #501
